packages/agent_sdk/runners/example_agent.py

```
@@ -0,0 +1,259 @@
+            )
+            
+            # Update message with results
+            message.next_actions = result.get("next_actions", [])
+            
+            return {
+                "status": "success",
+                "message": message.dict(),
+                "result": result
+            }
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)